iiod/dns-sd.c

```diff
@@ -255,7 +257,6 @@ static void start_avahi_thd(struct thread_pool *pool, void *d)
 
 	while(true) {
 		ret = gethostname(host, sizeof(host));
-		IIO_ERROR("host %s\n", host);
```

Comment:
If that's on purpose, it should be in a separate commit.

Reply:
yeah, OK - I think that was in there for debugging a long time ago, but I put it back...